wait/member.go

```diff
@@ -430,6 +432,19 @@ func (a *MemberAwaitility) UpdateIdlerSpec(idler *toolchainv1alpha1.Idler) (*too
 	return result, err
 }
 
+// Create tries to create the object until success
+// Workaround for https://github.com/kubernetes/kubernetes/issues/67761
+func (a *MemberAwaitility) Create(obj runtime.Object) error {
+	err := wait.Poll(a.RetryInterval, a.Timeout, func() (done bool, err error) {
+		if err := a.Client.Create(context.TODO(), obj); err != nil {
```

Comment:
I'm not sure but maybe we need to use a deep copy here (`obj.DeepCopy()`) instead of the original `obj`. We may need that if `Create()` actually modifies `obj` even if it failed with an error. Let's see how it works and if we see that issue we can start using a deep copy instead.